src/ui/profile_preferences.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>742</width>
-    <height>685</height>
+    <width>868</width>
+    <height>743</height>
```

Comment:
I've added some extra space because without it, opening the preferences tab now just presents you with a mass of tabs:

![workspace 1_104](https://user-images.githubusercontent.com/110988/27211089-191ecbec-5257-11e7-9c1b-b7d3cc1e74e8.png)

This is what I mean by there's no space for the IRC tab, the amount of tabs we have is approaching overwhelming.